### PR TITLE
chore: update client examples

### PR DESCRIPTION
**Pull request by vasco-santos**
Updates client examples. I will add tests for the examples as a follow up PR

TODO:

- [x] have `files-from-path` exported in the client https://github.com/web3-storage/web3.storage/pull/99

### CODE REVIEW

File: packages/client/examples/node.js/put-files.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import minimist from 'minimist'
+import { getFilesFromPath } from 'files-from-path'
+import { Web3Storage } from 'web3.storage'
+
+const defaultEndpoint = 'https://api.web3.storage' // the default
+
+async function main () {
+  const args = minimist(process.argv.slice(2))
+  const token = args.token
+  const endpoint = args.endpoint || defaultEndpoint
```
Comment (alanshaw) [inline packages/client/examples/node.js/put-files.js:11]: I would remove any endpoint stuff - just noise to new users.

File: packages/client/examples/browser/README.md [hunk 2]
```diff
@@ -1,8 +1,6 @@
 # Files demo browser - web3.storage
 
-🚧  **WORK IN PROGRESS**  🚧
-
-A demo using web3.storage client in the browser to pre-calculate the CID for an asset then storing it on tbd.storage and confirming that it uses the exact same CID for the asset.
+A demo using web3.storage client in the browser to pre-calculate the CID for an asset then storing it on web.storage.
```
Comment (alanshaw) [inline packages/client/examples/browser/README.md:3]:
```suggestion
A demo using web3.storage client in the browser to pre-calculate the CID for an asset then store it on web3.storage.
```

**Review by alanshaw — CHANGES_REQUESTED** [review]

File: packages/client/examples/node.js/package.json [hunk 3]
```diff
@@ -7,6 +7,11 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
   },
+  "dependencies": {
+    "files-from-path": "^0.1.0",
+    "minimist": "^1.2.5",
+    "web3.storage": "../.."
```
[the remark this answers is not in the document]
Comment (vasco-santos) [reply packages/client/examples/node.js/package.json:13]: Do you know if npm workspaces will deal with updating it? Changing it for now as you suggest

File: packages/client/examples/node.js/put-files-from-fs.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import process from 'process'
+import minimist from 'minimist'
+import { Web3Storage, getFilesFromPath } from 'web3.storage'
+
+const endpoint = 'https://api.web3.storage'
```
Comment (alanshaw) [inline packages/client/examples/node.js/put-files-from-fs.js:5]: Can you remove this as well please? Unnecessary for users to pass it.

```suggestion
```

File: packages/client/examples/node.js/put-files-from-fs.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+async function main () {
+  const args = minimist(process.argv.slice(2))
+  const token = args.token
+
+  if (!token) {
+    console.error('A token is needed. You can create one on https://web3.storage')
+    return
+  }
+
+  const storage = new Web3Storage({ endpoint, token })
```
Comment (alanshaw) [inline packages/client/examples/node.js/put-files-from-fs.js:16]:
```suggestion
  const storage = new Web3Storage({ token })
```

File: packages/client/examples/node.js/put-files.js [hunk 6]
```diff
@@ -1,22 +1,31 @@
-import { Web3Storage, File } from '../../src/lib.js'
+import process from 'process'
+import minimist from 'minimist'
+import { Web3Storage, File } from 'web3.storage'
 
-// TODO
-const endpoint = 'https://api.web3.storage' // the default
-const token = 'API_KEY' // your API key from https://web3.storage/manage
+const endpoint = 'https://api.web3.storage'
```
Comment (alanshaw) [inline packages/client/examples/node.js/put-files.js:5]:
```suggestion
```

File: packages/client/examples/node.js/put-files.js [hunk 7]
```diff
@@ -1,22 +1,31 @@
...
 async function main () {
+  const args = minimist(process.argv.slice(2))
+  const token = args.token
+
+  if (!token) {
+    console.error('A token is needed. You can create one on https://web3.storage')
+    return
+  }
+
   const storage = new Web3Storage({ endpoint, token })
```
Comment (alanshaw) [inline packages/client/examples/node.js/put-files.js:16]:
```suggestion
  const storage = new Web3Storage({ token })
```

**Review by alanshaw — CHANGES_REQUESTED** [review]

**Review by alanshaw — APPROVED** [review]

**Pull request merged**